providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdOptions.java

```diff
@@ -135,31 +130,14 @@ public class FlagdOptions {
      * Overload default lombok builder.
      */
     public static class FlagdOptionsBuilder {
-
-        /**
-         * File source of flags to be used by offline mode.
-         * Setting this enables the offline mode of the in-process provider.
-         */
-        public FlagdOptionsBuilder offlineFlagSourcePath(@Nonnull final String offlineFlagSourcePath) {
-            this.isOffline = true;
-            this.offlineFlagSourcePath = offlineFlagSourcePath;
-
-            return this;
-        }
-
-        // Remove the public access as this needs to be connected to offlineFlagSourcePath
-        @SuppressWarnings({"PMD.UnusedFormalParameter", "PMD.UnusedPrivateMethod"})
-        private FlagdOptionsBuilder isOffline(final boolean isOffline) {
-            return this;
-        }
-
         /**
          * Enable OpenTelemetry instance extraction from GlobalOpenTelemetry. Note that, this is only useful if global
          * configurations are registered.
          */
         public FlagdOptionsBuilder withGlobalTelemetry(final boolean b) {
-            this.openTelemetry = GlobalOpenTelemetry.get();
-
+            if (b) {
```

Comment:
🤕 how did we miss seeing this? nice catch :shipit:

Reply:
I saw this when cleaning the builder logic 😿